builders/integrations/indexers/subsquid.md

```diff
@@ -38,37 +38,43 @@ To get started indexing Substrate data on Moonbeam, you'll need to create a Subs
 
     For more information on getting started with this template, please check out the [Quickstart: Substrate chains](https://docs.subsquid.io/quickstart/quickstart-substrate/){target=_blank} guide on Subsquid's documentation site.
 
-2. To configure your Subsquid project to run on Moonbeam, you'll need to update the `typegen.json` file. The `typegen.json` file is responsible for generating TypeScript interface classes for your data. Depending on the network you're indexing data on, the `specVersions` value in the `typegen.json` file should be configured as follows:
+2. Navigate into the root directory of your Squid project and install dependencies by running:  
```

Comment:
Sometimes Squid is capitalized, and sometimes it isn't. Capitalizing it probably makes the most sense. Can you please capitalize it in the other instances?